src/FSharp.Build/Microsoft.FSharp.Targets

```diff
@@ -57,19 +57,27 @@ this file.
       <FSharpPreferNetFrameworkTools>  boolean: true or false === default value true
 
       Suggest that the compiler used be the 64 Bit compiler.  On computers without Visual Studio this property is ignored.
+      <FSharpPreferAnyCpuTools>      boolean: true or false === default value true
       <FSharpPrefer64BitTools>      boolean: true or false === default value true
 
-      If Shim helpers are present then the values are valid and set by the Microsoft.FSharp.Helpers.props
+      These are stupidly complex for performance reasons:
+        Arm64 is faster than AnyCpu on an Arm64 machine
+
+      On Windows Arm64 default to Arm64 build, otherwise default to AnyCpu.
```

Comment:
Interesting! Could it be also faster on macOS aarch64?

Reply:
Hm, good point. Should be.